### ISSUE TITLE
Wrap cypress helpers into js library for reuse across apps

### ISSUE THREAD

**Issue by max-nextcloud** [nextcloud/text#2201]
Wrap all those nice helpers into a small js library, so that they could be reused across apps when writing cypress tests.

_Originally posted by @juliushaertl in https://github.com/nextcloud/text/pull/2193#r812626438_

**Comment by mejo-** [comment]
@max-nextcloud is this rather about moving some of our custom cypress commands to `@nextcloud/cypress`?

**Comment by max-nextcloud** [comment]
Yes. But given that more and more apps are using playwright it might not be worth the effort. I'd say let's close this and move helpers if we notice we keep copying them to other apps.

**Comment by mejo-** [comment]
I agree!
